service/src/event/ordering_task.rs

```diff
@@ -206,9 +210,13 @@ impl StreamEvents {
         Self::default()
     }
 
-    /// returns true if we didn't know about this event previously
-    pub fn add_event(&mut self, event: DeliverableEvent) -> bool {
-        let res = self.prev_map.insert(event.meta.prev, event.cid).is_some();
+    /// returns Some(Stream init CID) if we didn't know about this event previously
+    pub fn add_event(&mut self, event: DeliverableEvent) -> Option<InitCid> {
+        let res = if self.prev_map.insert(event.meta.prev, event.cid).is_none() {
```

Comment:
I had the `is_some()`/`is_none()` backward before 😞 but I was only using it for some log messages and not relying on it for anything